src/siirtc.c

```diff
@@ -98,8 +103,12 @@ u8 SiiRtcProbe(void)
 
     errorCode = 0;
 
+#ifndef BUGFIX
```

Comment:
What is the bug here?

Reply:
If both power failure happened and the clock was 12 hour, the clock wouldn’t be reset